downloads/id/github-git-cheat-sheet.md

```diff
@@ -122,22 +126,22 @@ Menghapus berkas dari riwayat revisi dengan tetap mempertahankan berkas lokal
 
 Mengganti nama berkas dan mempersiapkan berkas untuk pendaftaran _commit_
 
-## Tahan pelacakan
-Mengabaikan berkas dan jalur untuk sementara
+## Tahan Pelacakan
+Mengabaikan berkas dan garis edar sementara
 
 ```
 *.log
 build/
 temp-*
 ```
 
-Sebuah berkas teks bernama `.gitignore` mengabaikan revisi berkas yang tidak disengaja serta jalur berkas yang cocok dengan pola tertentu
+Sebuah berkas teks bernama `.gitignore` mengabaikan revisi berkas yang tidak disengaja serta garis edar berkas yang cocok dengan pola tertentu
 
 ```$ git ls-files --others --ignored --exclude-standard```
 
-Daftarkan semua berkas yang diabaikan dalam proyek ini
+Daftar semua berkas yang diabaikan dalam proyek tersebut
```

Comment:
Menampilkan daftar semua berkas yang diabaikan dalam proyek tersebut because we need  Verbs